File: Iniciante/1048.py
S = float(input())

if(S >= 0 and S <= 400.00):
    PR = (15 * S) / 100
    print(f'Novo salario: {S+PR:.2f}\nReajuste ganho: {PR:.2f}\nEm percentual: 15 %')
elif(S >= 400.01 and S <= 800.00):
    PR = (12 * S) / 100
    print(f'Novo salario: {S+PR:.2f}\nReajuste ganho: {PR:.2f}\nEm percentual: 12 %')
elif(S >= 800.01 and S <= 1200.00):
    PR = (10 * S) / 100
    print(f'Novo salario: {S+PR:.2f}\nReajuste ganho: {PR:.2f}\nEm percentual: 10 %')
elif(S >= 1200.01 and S <= 2000.00):
    PR = (7 * S) / 100
    print(f'Novo salario: {S+PR:.2f}\nReajuste ganho: {PR:.2f}\nEm percentual: 7 %')
elif(S > 2000.00):
    PR = (4 * S) / 100
    print(f'Novo salario: {S+PR:.2f}\nReajuste ganho: {PR:.2f}\nEm percentual: 4 %')

File: Iniciante/1048.php
<?php

$s = readline();

if($s >= 0 && $s <= 400.00){
    $per = (15 * $s) / 100;
    echo "Novo salario: ".number_format($s+$per, 2, '.', '')."\nReajuste ganho: ".number_format($per, 2, '.', '')."\nEm percentual: 15 %\n";
}elseif($s >= 400.01 && $s <= 800.00){
    $per = (12 * $s) / 100;
    echo "Novo salario: ".number_format($s+$per, 2, '.', '')."\nReajuste ganho: ".number_format($per, 2, '.', '')."\nEm percentual: 12 %\n";
}elseif($s >= 800.01 && $s <= 1200.00){
    $per = (10 * $s) / 100;
    echo "Novo salario: ".number_format($s+$per, 2, '.', '')."\nReajuste ganho: ".number_format($per, 2, '.', '')."\nEm percentual: 10 %\n";
}elseif($s >= 1200.01 && $s <= 2000.00){
    $per = (7 * $s) / 100;
    echo "Novo salario: ".number_format($s+$per, 2, '.', '')."\nReajuste ganho: ".number_format($per, 2, '.', '')."\nEm percentual: 7 %\n";
}elseif($s > 2000.00){
    $per = (4 * $s) / 100;
    echo "Novo salario: ".number_format($s+$per, 2, '.', '')."\nReajuste ganho: ".number_format($per, 2, '.', '')."\nEm percentual: 4 %\n";
}

?>